.env-dist

```diff
@@ -2,3 +2,5 @@ DEBUG=True
 DEV=True
 ALLOWED_HOSTS=*
 SWITCH_NEWSLETTER_MAINTENANCE_MODE=False
+CSP_DEFAULT_SRC=*.allizom.org
```

Comment:
Hmm, when running locally (not using Docker) I still see CSP issues e.g.

```
Content Security Policy: The page's settings blocked the loading of a resource at http://localhost:8000/media/js/BUNDLES/site.js ("script-src").
```

Reply:
@alexgibson it's working for me. I honestly have no idea. Does the `CSP_DEFAULT_SRC` setting in your `.env` file look exactly like this line?

Reply:
I tried this again with a fresh `.env` and it worked both locally and using docker. Not sure what I did wrong the first time, but I can't seem to replicate this error any longer? :)